core-bundle/contao/templates/forms/form_wrapper.html5

```diff
@@ -6,19 +6,15 @@
     <<?= $this->hl ?>><?= $this->headline ?></<?= $this->hl ?>>
   <?php endif; ?>
 
-  <form<?php if ($this->action): ?> action="<?= $this->action ?>"<?php endif; ?> method="<?= $this->method ?>" enctype="<?= $this->enctype ?>"<?= $this->attributes ?><?= $this->novalidate ?>>
-    <div class="formbody">
-      <?php if ('get' != $this->method): ?>
-        <input type="hidden" name="FORM_SUBMIT" value="<?= $this->formSubmit ?>">
-        <input type="hidden" name="REQUEST_TOKEN" value="<?= $this->requestToken ?>">
-        <?php if ($this->maxFileSize): ?>
-          <input type="hidden" name="MAX_FILE_SIZE" value="<?= $this->maxFileSize ?>">
-        <?php endif; ?>
-      <?php endif; ?>
-      <?= $this->hidden ?>
-      <?= $this->fields ?>
-    </div>
-  </form>
+  <?php if ($this->confirmation): ?>
+    <?php $this->insert('form_confirmation', $this->arrData) ?>
+  <?php else: ?>
+    <?php $this->insert('form_inline', $this->arrData) ?>
+  <?php endif; ?>
+
+  <?php if ($this->ajax): ?>
+    <script src="<?= $this->asset('ajax-form.min.js', 'contao_core') ?>"></script>
```

Comment:
This is not ideal because if there are several forms on the page, the script will be loaded and executed multiple times. It should be added to `$GLOBALS['TL_BODY']` instead.

Reply:
I have added it to `$GLOBALS['TL_JAVASCRIPT']`.